css/src/components/breadcrumbs.css

```diff
@@ -32,15 +26,19 @@
 
 .Breadcrumbs-itemSeparator {
   color: var(--secondary);
+  font-size: var(--font-size-s);
+  line-height: var(--font-height-s);
+  width: var(--spacing-m);
+  margin: 0 var(--spacing-m);
 }
 
 .Breadcrumbs-dropdown {
-  display: flex;
-  align-items: center;
+  height: var(--font-height-m);
 }
 
-.Breadcrumbs-Button {
-  margin: 0 var(--spacing-m);
+.Breadcrumbs-collapse {
+  display: flex;
+  align-items: center;
```

Comment:
Instead of creating new class please use `d-flex align-items-center` directly

Reply:
Done